test/cmd/kubeadm/token_test.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubeadm
```

Comment:
Thanks for doing this. Great!

However, I think it would be better if this was under `cmd/kubeadm/test` instead, can you do that?
In that case, we have everything centralized under `cmd/kubeadm` where we have the right OWNERS/assignees etc.